### ISSUE TITLE
Add "Warning/Tip" to top of older Release Notes that they might have outdated info

### ISSUE THREAD

**Issue by NotTheDr01ds** [nushell/nushell.github.io#1501]
As mentioned #536, search results can surface some pretty old Release Notes that have outdated info.  A couple of options:

1. Leave it as it is
2. Review each release note and edit to highlight outdated info
3. Just add a `warning` header (yellow formatting in the theme) to the top of older (say 5 releases back and older) release notes with something like (first pass):

   > These are the Release Notes for `<version>`.  If you have arrived here via Search, please keep in mind that information here may (or may not ) be outdated.  Features may have been renamed, functionality added or removed, or other changes may have been made after this Release. Please refer to the other documentation sources on this site for updated information.

I'd vote for the third, personally.  Regarding 2 - I think Release Notes are a historical archive and shouldn't be changed post de defacto.  It would also be a major (and continual) undertaking to review and update.

While the first option is a possibility, the point made in #536 is a fair one that I'd like to see addressed in some say.

**Comment by Kissaki** [comment]
I agree. Adding a may-be- or is-probably-outdated notice seems like the most sensible solution.

Notably, links may jump to a point further down in release notes. That means a warning at the top may not be obvious.

Using a sticky warning may be a solution; that always displays even when scrolling away from the top.
